eslint.config.js

```diff
@@ -1,5 +1,167 @@
 import blumilkDefault from '@blumilksoftware/eslint-config'
```

Comment:
When you want to update linter rules, it will be great to update some dependencies. For example  `@blumilksoftware/eslint-config` to new version.